prod/westeurope/functions_app1/functions_app1_r3/function_app/terragrunt.hcl

```diff
@@ -0,0 +1,205 @@
+
+    // Push notifications
+    AZURE_NH_HUB_NAME                       = dependency.notification_hub.outputs.name
+    NOTIFICATIONS_QUEUE_NAME                = dependency.notification_queue.outputs.name
+    NOTIFICATIONS_STORAGE_CONNECTION_STRING = dependency.notification_storage_account.outputs.primary_connection_string
+
+    SLOT_TASK_HUBNAME = "ProductionTaskHub"
+
+    WEBSITE_VNET_ROUTE_ALL = 1
+
+    // Disable functions
+    #"AzureWebJobs.CreateProfile.Disabled"                          = "1"
+    #"AzureWebJobs.CreateValidationTokenActivity.Disabled"          = "1"
+    #"AzureWebJobs.EmailValidationProcessOrchestrator.Disabled"     = "1"
+    #"AzureWebJobs.GetMessage.Disabled"                             = "1"
+    #"AzureWebJobs.GetMessages.Disabled"                            = "1"
+    #"AzureWebJobs.GetProfile.Disabled"                             = "1"
+    #"AzureWebJobs.GetService.Disabled"                             = "1"
+    #"AzureWebJobs.GetUserDataProcessing.Disabled"                  = "1"
+    #"AzureWebJobs.GetVisibleServices.Disabled"                     = "1"
+    #"AzureWebJobs.HandleNHNotificationCall.Disabled"               = "1"
+    #"AzureWebJobs.HandleNHNotificationCallActivity.Disabled"       = "1"
+    #"AzureWebJobs.HandleNHNotificationCallOrchestrator.Disabled"   = "1"
+    #"AzureWebJobs.SendUserDataProcessingEmailActivity.Disabled"    = "1"
+    #"AzureWebJobs.SendValidationEmailActivity.Disabled"            = "1"
+    #"AzureWebJobs.SendWelcomeMessagesActivity.Disabled"            = "1"
+    #"AzureWebJobs.StartEmailValidationProcess.Disabled"            = "1"
+    #"AzureWebJobs.StoreSpidLogs.Disabled"                          = "1"
+    #"AzureWebJobs.UpdateProfile.Disabled"                          = "1"
+    #"AzureWebJobs.UpdateSubscriptionsFeedActivity.Disabled"        = "1"
+    #"AzureWebJobs.UpsertUserDataProcessing.Disabled"               = "1"
+    #"AzureWebJobs.UpsertedProfileOrchestrator.Disabled"            = "1"
+    #"AzureWebJobs.UpsertedUserDataProcessingOrchestrator.Disabled" = "1"
+    "AzureWebJobs.HandleNHNotificationCall.Disabled" = "1"
+    "AzureWebJobs.StoreSpidLogs.Disabled"            = "1"
+
+    # Cashback
+    IS_CASHBACK_ENABLED = "true"
+  }
+
+  app_settings_secrets = {
+    key_vault_id = dependency.key_vault.outputs.id
+    map = {
+      __DISABLED__SENDGRID_API_KEY = "common-SENDGRID-APIKEY"
+      MAILUP_USERNAME              = "common-MAILUP2-USERNAME"
+      MAILUP_SECRET                = "common-MAILUP2-SECRET"
+      PUBLIC_API_KEY               = "apim-IO-SERVICE-KEY"
+      SPID_LOGS_PUBLIC_KEY         = "funcapp-KEY-SPIDLOGS-PUB"
+      AZURE_NH_ENDPOINT            = "common-AZURE-NH-ENDPOINT"
+    }
+  }
+
+  allowed_subnets = [
+    dependency.subnet.outputs.id,
+    dependency.subnet_appbackend.outputs.id,
+    dependency.subnet_appbackend_l1.outputs.id,
+    dependency.subnet_appbackend_l2.outputs.id,
```

Comment:
```suggestion
```
if the link will be appbackend_l1 -> fn3app1, subnet_appbackend_l2 is not necessary